[2023-09-15T06:46:32.141+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ganza_dag_v13.greet scheduled__2023-09-14T00:00:00+00:00 [queued]>
[2023-09-15T06:46:32.154+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ganza_dag_v13.greet scheduled__2023-09-14T00:00:00+00:00 [queued]>
[2023-09-15T06:46:32.154+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-15T06:46:32.183+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): greet> on 2023-09-14 00:00:00+00:00
[2023-09-15T06:46:32.187+0000] {standard_task_runner.py:57} INFO - Started process 213 to run task
[2023-09-15T06:46:32.195+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ganza_dag_v13', 'greet', 'scheduled__2023-09-14T00:00:00+00:00', '--job-id', '309', '--raw', '--subdir', 'DAGS_FOLDER/my_second_dag.py', '--cfg-path', '/tmp/tmp1v09m12a']
[2023-09-15T06:46:32.201+0000] {standard_task_runner.py:85} INFO - Job 309: Subtask greet
[2023-09-15T06:46:32.271+0000] {task_command.py:415} INFO - Running <TaskInstance: ganza_dag_v13.greet scheduled__2023-09-14T00:00:00+00:00 [running]> on host 2729c554a115
[2023-09-15T06:46:32.422+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ganza_m' AIRFLOW_CTX_DAG_ID='ganza_dag_v13' AIRFLOW_CTX_TASK_ID='greet' AIRFLOW_CTX_EXECUTION_DATE='2023-09-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-14T00:00:00+00:00'
[2023-09-15T06:46:32.459+0000] {logging_mixin.py:151} INFO - hello world, My Name is gmx the DJ,and I am 20 years old!
[2023-09-15T06:46:32.460+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-09-15T06:46:32.473+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ganza_dag_v13, task_id=greet, execution_date=20230914T000000, start_date=20230915T064632, end_date=20230915T064632
[2023-09-15T06:46:32.532+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-15T06:46:32.557+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
